DependencyInjection/Configuration.php

```diff
@@ -397,6 +314,41 @@ private function addProxyClientSection(ArrayNodeDefinition $rootNode)
             ->end();
     }
 
+    /**
+     * Get the configuration node for a HTTP dispatcher in a proxy client.
+     *
+     * @return NodeDefinition
+     */
+    private function getHttpDispatcherNode()
+    {
+        $treeBuilder = new TreeBuilder();
+        $node = $treeBuilder->root('http');
+
+        $node
+            ->fixXmlConfig('server')
+            ->isRequired()
+            ->children()
+                ->arrayNode('servers')
+                    ->info('Addresses of the hosts the caching proxy is running on. May be hostname or ip, and with :port if not the default port 80.')
+                    ->useAttributeAsKey('name')
+                    ->isRequired()
+                    ->requiresAtLeastOneElement()
+                    ->prototype('scalar')->end()
+                ->end()
+                ->scalarNode('base_url')
+                    ->defaultNull()
+                    ->info('Default host name and optional path for path based invalidation.')
+                ->end()
+                ->scalarNode('http_client')
+                    ->defaultNull()
```

Comment:
we don't need any reference to the httplug-bundle. by default, we pass no client to HttpDispatcher, which will prompt that to do a discovery. if discovery is not desired, a client can be configured.